data/avgi/avgi 1 first contact.txt

```diff
@@ -0,0 +1,1055 @@
+			`	He looks down at a device similar to a tablet. "With the attack on Ensemble, we were forced to pull back our main fleetgroups from the front lines in Quintessence in an attempt to contain them. While we have since reclaimed many of our initial positions, during all the maneuvering a large number of Aberrant were able to slip past us, not into the Shroud, but rather north, towards the Empyrean system. With the current situation, I do not want to divert an entire strike force from the defenses to hunt them down. Report back to the Feo Platform when the Aberrant are eliminated."`
+				accept
+	on accept
+		log `Agreed to help Kokkino by clearing out Aberrant stragglers infesting a cluster of systems behind the front lines.`
+
+	npc kill
+		government "Aberrant"
+		personality
+			confusion 40
+			coward secretive staying target unconstrained uninterested vindictive
+		system "Corporeal"
+		fleet
+			names "aberrant names"
+			variant
+				"Aberrant Junior"
+				"Aberrant Icebreaker"
+	npc kill
+		government "Aberrant"
+		personality
+			confusion 40
+			coward secretive staying target unconstrained uninterested vindictive
+		system "Empyrean"
+		fleet
+			names "aberrant names"
+			variant
+				"Aberrant Hugger"
+				"Aberrant Pileup"
+	npc kill
+		government "Aberrant"
+		personality
+			confusion 40
+			coward secretive staying target unconstrained uninterested vindictive
+		system "Firmament"
+		fleet
+			names "aberrant names"
+			variant
+				"Aberrant Triplet"
+	npc kill
+		government "Aberrant"
+		personality
+			confusion 40
+			coward secretive staying target unconstrained uninterested vindictive
+		system "Lumine"
+		fleet
+			names "aberrant names"
+			variant
+				"Aberrant Chomper" 2
+
+	on complete
+		conversation
+			`You receive a hero's welcome back on the Feo Platform from the Twilight Guards stationed there. After a short celebration and many expressions of praise toward you, an alien stranger, you find Darius and Sora waiting by the <ship> to speak with you.`
+			`	"<first>, I think we have a golden opportunity," Darius says. "I was just talking to an Assemblyman, Hriso, who's worried about the political situation in the eastern branch of the Tangled Shroud."`
+			`	"A bunch of selfish cowards, they are," says Sora disdainfully.`
+			`	"Something about the miners grumbling about price controls on nickel-iron," Darius says after a moment, "but he wants to know if you could use your jump drive to take him there for some outreach to the miners without needing to spend weeks wandering through the mess of twisted hyperlanes. It's the perfect chance to meet my friend and ask him some questions. We'll need to head to Peripheria to pick up Hriso and get this trip sorted out."`
```

Comment:
This should also be in the "on offer" of the next mission, especially because that mission requires passenger space and this does not, so a person might see this conversation but not have a new mission.